docs/reference/api/pages.md

```diff
@@ -1,33 +1,37 @@
 # Page type REST endpoints
 
-All [page types](#TODO) use a single set of API endpoints, unlike piece types. Difference in page type will primarily influence returned document properties based on the page type configurations.
+All [page types](/reference/glossary.md#page) use a single set of API endpoints, unlike piece types. Difference in page type will primarily influence returned document properties based on the page type configurations.
 
 ## Endpoints
 
+[Authentication](/reference/api/authentication.md) is required for all REST API requests.
+
 ### REST endpoints
 
-| Method | Path | Description | Auth required |
-|---------|---------|---------|---------|
-|`GET` | [`/api/v1/@apostrophecms/page` ](#get-api-v1-apostrophecms-page)| Get the home page and all other pages structured in the home page's `_children` property | FALSE |
```

Comment:
These big change blocks for the tables are from removing the "Auth required" column since it's required for all.